nix/sources.json

```diff
@@ -41,22 +41,10 @@
         "homepage": null,
         "owner": "input-output-hk",
         "repo": "iohk-nix",
-        "rev": "83109208047b07d5f0c103c87e6685c61c36a9f6",
-        "sha256": "1wfflfhd3jc8j49kgbvw0absvibkwm77wrjjc9ls8xkrcnprn4l7",
+        "rev": "72262b212e0cd8efe119466f0cb123f9d3760d35",
```

Comment:
Would you be able to use latest master, which contains a commitIdFromGitRepo improvement?

Reply:
done.